<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="Checkers_Board">
    <static>true</static>
    <link name="checkers_board">
      <pose>0 0 0.0025 0 0 0</pose>
      <collision name="collision">
        <geometry>
          <box>
            <size>.2921 0.2921 .005</size>
          </box>
        </geometry>
        <surface>
            <friction>
            <ode>
                <mu>0.01</mu>
                <mu2>0.01</mu2>
            </ode>
            </friction>
        </surface>
      </collision>
      <visual name="visual">
        <cast_shadows>false</cast_shadows>
        <geometry>
          <box>
            <size>.2921 0.2921 .005</size>
          </box>
        </geometry>
        <material>
            <script>
                <uri>model://checkers_board/materials/scripts</uri>
                <uri>model://checkers_board/materials/textures</uri>
                <name>Checkers/Board</name>
            </script>
        </material>
      </visual>
     <inertial>
        <mass>0.068</mass>
        <inertia> 
          <ixx>0.00048</ixx>       
          <ixy>0.0</ixy>         
          <ixz>0.0</ixz>        
          <iyy>0.00048</iyy>       
          <iyz>0.0</iyz>         
          <izz>0.001</izz>       
        </inertia>
      </inertial>
    </link>
  </model>


</sdf>